ansible/roles/airflow/tasks/af-container.yml

```diff
@@ -0,0 +1,36 @@
+---
+- file: path=/srv/{{ dockname }} state=directory mode=0750 owner={{ login }} group={{ login }}
+- stat: path=/srv/{{ dockname }}/airflow.cfg
+  register: afcfg
+- command: cp -a /srv/etc/af-share-{{ airflow_tag }}/airflow.cfg /srv/{{ dockname }}/airflow.cfg
```

Comment:
That's true, I was lazy debater :)

Maybe `-a` was the reason. I'll think a bit about it and, if `-a` is not the reason I'll replace it with `copy`.